ios/.ruby-version

```diff
@@ -0,0 +1 @@
+3.1.0
```

Comment:
just adding a note as I noticed the other ones (android) were removed but not this one